code/src/components/Form.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from 'react'
+import SelectForm from './SelectForm.js'
+import InputForm from './InputForm.js'
+import CheckboxForm from './CheckboxForm.js'
+import RadioForm from './RadioForm.js'
+import ProgressBar from './ProgressBar.js'
+
+const Form = ({input, setInput, setSubmit}) => {
+    const [currentQuestion, setCurrentQuestion] = useState(0);
+    const [errorMessage, setErrorMessage] = useState('');
+
+    const componentsArray = [
+        <InputForm 
+            input={input}
+            setInput={setInput}
+            key={0}
+        />,
+        <SelectForm 
+            input={input}
+            setInput={setInput}
+            key={1}
+        />,
+        <CheckboxForm 
+            input={input}
+            setInput={setInput}
+            key={2}
+        />,
+        <RadioForm 
+            input={input}
+            setInput={setInput}
+            key={3}
```

Comment:
Wondering what the `key={0}` and so on.. is used for on lines 16, 21, 26, and 31?  I can't seem to see them being implemented in the InputForm, SelectForm, CheckboxForm, and RadioForm components.